content/300-accelerate/200-getting-started.mdx

```diff
@@ -195,3 +195,54 @@ For information about which strategy best serves your application, see [Select a
 As of Prisma version `5.2.0` you can use Prisma Studio with the Accelerate connection string.
 
 </Admonition>
+
+#### Invalidating a cached query result
+
+To invalidate a cached query result, you can add tags and then use the `$accelerate.invalidate` API.
+
+:::note
+
+Programmatic cache invalidation is available with our paid plans. For more details, please see our [pricing](https://www.prisma.io/pricing#accelerate).
+
+:::
+
+To invalidate the query below:
+
+```ts
+await prisma.user.findMany({
+  where: {
+    email: {
+      contains: "alice@prisma.io",
+    },
+  },
+  cacheStrategy: {
+    swr: 60,
+    ttl: 60,
+    // highlight-start
+    tags: ["emails_with_alice"],
+    // highlight-end
+  },
+});
+```
+
+You need to provide the cache tag in the `$accelerate.invalidate` API:
```

Comment:
I think explaining wha the tag is here, "the tag is used as a target/name/variable" what ever the right term is to give persise control over invalidation of specific queries within your application 

(or something like that)

Reply:
I explained here and believe that's the right place as it's the API ref: /accelerate/api-reference#options